bindings/rust/aws-lc-sys-template/build.rs

```diff
@@ -0,0 +1,318 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+
+/* Copyright (c) 2022, Google Inc.
+ *
+ * Permission to use, copy, modify, and/or distribute this software for any
+ * purpose with or without fee is hereby granted, provided that the above
+ * copyright notice and this permission notice appear in all copies.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES
+ * WITH REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED WARRANTIES OF
+ * MERCHANTABILITY AND FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR ANY
+ * SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
+ * WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS, WHETHER IN AN ACTION
+ * OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF OR IN
+ * CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE. */
+
```

Comment:
This doesn't seem right, is this an original file with code from AWS or from Google?

Reply:
I started from the original [build.rs](https://github.com/awslabs/aws-lc/blob/main/rust/build.rs) and just never removed the original copyright notice.. Most of the code is either original or copied from [here](https://github.com/cloudflare/boring/blob/master/boring-sys/build.rs).

Reply:
I removed the code I had adapted from the other (MIT licensed) source.  All of the code in that file now is either original to that file or code I had written on my own.